### DISCUSSION TITLE
Is there a way to directly pass in html markup maybe using dangerouslySetInnerHTML?

### DISCUSSION THREAD

**Discussion by romellogoodman** [vercel/satori#164] [Q&A]
I'm thinking something along the lines of this

```
function handler() {
  const graphics = getSVG(conf, {});

  return new ImageResponse(
    {
      type: "div",
      props: {
        dangerouslySetInnerHTML: "<p>Your html code here.<p>",
      },
    },
    {
      width: 800,
      height: 800,
    }
  );
}

export default handler;
```

**Comment by shuding** [comment]
This doesn't work because Satori doesn't include a HTML parser. Maybe you can try something like https://github.com/developit/htm

**Comment by romellogoodman** [comment]
htm doesn't seem to work because it doesn't support svg at the moment (https://github.com/developit/htm/issues/82). I was able to get a solution working with  https://www.npmjs.com/package/html-to-react
